spec/dummy/db/schema.rb

```diff
@@ -10,11 +10,22 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2022_07_08_103727) do
+ActiveRecord::Schema.define(version: 2022_07_14_164313) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "ad_renewal_letters_exports", id: :serial, force: :cascade do |t|
```

Comment:
Think Ive had previous issues with ad_renew_letters_exports, is this meant to be in the schema? :)

Reply:
Good catch, I must have re-added it while trying to resolve a local migration issue. Removed now.